spaceship/spec/test_flight/client_spec.rb

```diff
@@ -126,14 +126,6 @@ def handle_response(response)
   # @!group Testers API
   ##
 
-  context '#testers_for_app' do
-    it 'executes the request' do
-      MockAPI::TestFlightServer.get('/testflight/v2/providers/fake-team-id/apps/some-app-id/testers') {}
-      subject.testers_for_app(app_id: app_id)
-      expect(WebMock).to have_requested(:get, 'https://itunesconnect.apple.com/testflight/v2/providers/fake-team-id/apps/some-app-id/testers?limit=10000')
-    end
-  end
-
```

Comment:
How come we are removing a test here? Don't we need to test that the new paging thingy works?

Reply:
It's part of the PR body, the test doesn't seem to test anything really

Reply:
Yep, could be adding new tests

Reply:
I am confused. This test ensures that the method `testers_for_app(...)` makes a GET request to a specific URL.